css/subject_viewer.styl

```diff
@@ -370,6 +370,15 @@
     .followup {
       display: block;
     }
+
+    .annotations .annotation-item button {
+      visibility: hidden;
```

Comment:
`visibility: hidden` is redundant if you have `display: none` below; the former hides the box but leaves the space, but the latter then hides the space